string class_name
float32 probability